internal/pkg/users/users.go

```diff
@@ -16,26 +16,24 @@ limitations under the License.
 package users
 
 import (
+	"os/exec"
 	"os/user"
 	"strconv"
+	"strings"
 )
 
 // GetUID returns uid of given username and logs a warning if its missing
 func GetUID(name string) (int, error) {
 	entry, err := user.Lookup(name)
-	if err != nil {
-		return 0, err
+	if err == nil {
+		return strconv.Atoi(entry.Uid)
 	}
-	return strconv.Atoi(entry.Uid)
-}
-
-func CheckIfUserExists(name string) (bool, error) {
-	_, err := user.Lookup(name)
 	if _, ok := err.(user.UnknownUserError); ok {
-		return false, nil
-	}
-	if err != nil {
-		return false, err
+		// fallback to call external `id` in case NSS is used
+		out, err := exec.Command("id", "-u", name).CombinedOutput()
```

Comment:
Can this be the full path to `id` instead?

Reply:
It can.